crates/core/thin_edge_json/src/utils.rs

```diff
@@ -48,3 +20,29 @@ fn excerpt_returns_string_starting_from_line_and_column() {
     );
     assert_eq!("n", excerpt("line 1\nline 2\nline 3\n", 2, 3, 1));
 }
+
+#[test]
+fn excerpt_returns_string_starting_first_line_and_column() {
+    assert_eq!("line 1", excerpt("line 1\nline 2\nline 3\n", 1, 1, 6));
```

Comment:
The indexing starting with `1` instead of `0` was a bit confusing at first, but then got used to it. A bit curious why this "natural" indexing was used here instead of the "programmer-friendly" one.

Reply:
This natural indexing is imposed by `serde`. I guess this is because these errors messages are for the end-user.